jdi-light/src/main/java/com/epam/jdi/light/elements/common/UIElement.java

```diff
@@ -619,7 +619,7 @@ protected boolean selected() {
     }
     protected boolean enabled() {
         List<String> cls = classes();
-        return cls.contains("active") || !hasAttribute("disabled") || getWebElement().isEnabled() && !cls.contains("disabled");
+        return cls.contains("active") || !hasAttribute("disabled") && !hasClass("disabled") || getWebElement().isEnabled() && !cls.contains("disabled");
```

Comment:
@RomanIovlev  we need your approve here

Reply:
we already have part: !cls.contains("disabled");
why do we need standalone !hasClass("disabled") ?

Reply:
The part you've mentioned doesn't yield the expected result. 
The initial code checks for the presence of the "disabled" attribute, not the "disabled" class - that was the problem with the initial implementation.

Reply:
cls contains classes, isn't it? so if !cls.contains("disabled"); is not working as we expected, we should reserch and fix it. because !cls.contains("disabled") is the same !hasClass("disabled") . or we just should to leave !hasClass("disabled")  and remove !cls.contains("disabled") as obsolete (but if it is not working we still must to fix it)

Reply:
Okay Imma do some research on that one now then

Reply:
Done. Please check the modified version.